sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # Release History
 
-## 3.0.0 (Unreleased)
+## 2.3.1 (Unreleased)
```

Comment:
```suggestion
## 2.3.0 (Unreleased)
```

Reply:
ci/cd build fails though because its still expecting a 3.0.0 in the change log.